views/tracks.hbs

```diff
@@ -0,0 +1,22 @@
+<div class="container-fluid ">
+    <div class="row">
+        <div class="col-12 text-center">
+            <h1>Tracks</h1>
+            <ul>
+                {{#each tracks}}
+                <li>
+                    {{this.name}}
```

Comment:
It's better to have `{{this.name}}` inside a `<span>` in case you want to style it later (or any other elements you feel is appropriate).